package.json

```diff
@@ -9,16 +9,16 @@
     "acquit-ignore": "0.1.0",
     "acquit-markdown": "0.1.0",
     "mocha": "5.x",
-    "mongodb": "3.x",
-    "mongoose": "5.x"
+    "mongodb": "4.x",
+    "mongoose": ">=5.12.10"
   },
   "engines": {
     "node": ">= 4.0.0"
   },
   "license": "Apache 2.0",
   "main": "int32.js",
   "peerDependencies": {
-    "mongoose": ">= 4.4.0 || 5.x"
```

Comment:
Since cast error message was changed in `mongoose@5.12.10` I thought that we should have mongoose 4 as peer dependency. Lmk if it's fine to have v4 here as well.